src/Engine/ProtoCore/Lang/BuiltInFunctionEndPoint.cs

```diff
@@ -497,6 +499,7 @@ public override StackValue Execute(ProtoCore.Runtime.Context c, List<StackValue>
                         if (array.IsArray)
                         {
                             bool result = runtimeCore.Heap.ToHeapObject<DSArray>(array).ContainsKey(key);
+                            runtimeCore.RuntimeStatus.LogWarning(WarningID.Default, "\"ContainsKey\" is deprecated, please use Dictionary data type with Dictionary.Keys & List.Contains");
```

Comment:
can you stick these strings in the resx file?

Reply:
How important is this?

Reply:
theres a check box. I make external devs do it.

Reply:
@saintentropy we can merge this so it can be tested and then add the resx files as a sep PR.

Reply:
localization strings added :-)